packages/jest-config/src/Defaults.ts

```diff
@@ -55,15 +55,16 @@ const defaultOptions: Config.DefaultOptions = {
   slowTestThreshold: 5,
   snapshotSerializers: [],
   testEnvironment: 'jest-environment-jsdom',
-  testEnvironmentOptions: {},
+  testEnvironmentOptions: {
+    url: 'http://localhost',
+  },
```

Comment:
The default `testEnvironment` is now "node" and we don't want set `testEnvironmentOptions` for that. Please rebase to latest master to see the change

Reply:
ooh, good point... Would be nice to default to that URL for jsdom, tho... Should we default in the env itself rather than in config?

Reply:
@SimenB @thymikee  pls how can i go around this

Reply:
@SimenB @thymikee does this help in any way to default URL to jsdom 
 ```
if (newOptions.testEnvironment.includes('jest-environment-jsdom')) {
    newOptions.testEnvironmentOptions = {
      url: newOptions.testEnvironmentOptions.url || 'http://localhost',
    };
  }
```
am also facing this error :  the jsdom console is the same as the test console

    expect(received).toBe(expected) // Object.is equality

    Expected: 0
    Received: 1      const {summary, rest} = extractSummary(stderr);
      93 | 
    > 94 |   expect(exitCode).toBe(0);
         |                    ^
      95 |   expect(wrap(stdout)).toMatchSnapshot();
      96 |   expect(wrap(rest)).toMatchSnapshot();
      97 |   expect(wrap(summary)).toMatchSnapshot();`

Reply:
and also when the `testURL` is removed , how should i handle this, since i cant set url at default , if am getting it wrong pls kindly lead me through
 ```
 this.dom = new JSDOM('<!DOCTYPE html>', {
      pretendToBeVisual: true,
      runScripts: 'dangerously',
      url: config.testURL, //'http://localhost' 
      virtualConsole: new VirtualConsole().sendTo(options.console || console),
      ...config.testEnvironmentOptions,
    });
```

Reply:
url will be inside `config.testEnvironmentOptions` so just remove it